mysql-test/suite/rpl_encryption/r/encrypted_slave.result

```diff
@@ -145,6 +145,8 @@ ANALYZE TABLE myisam_table_name_to_encrypt;
 CHECK TABLE innodb_table_name_to_encrypt_2;
 CHECKSUM TABLE innodb_table_name_to_encrypt, myisam_table_name_to_encrypt;
 RENAME USER user_name_to_encrypt to new_user_name_to_encrypt;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
As above.